aiida/backends/tests/cmdline/commands/test_import.py

```diff
@@ -85,8 +89,81 @@ def test_comment_mode(self):
         result = self.cli_runner.invoke(cmd_import.cmd_import, options)
         self.assertIsNone(result.exception, result.output)
         self.assertIn('Comment mode: newest', result.output)
+        self.assertEqual(result.exit_code, 0, result.output)
 
         options = ['--comment-mode', 'overwrite'] + archives
         result = self.cli_runner.invoke(cmd_import.cmd_import, options)
         self.assertIsNone(result.exception, result.output)
         self.assertIn('Comment mode: overwrite', result.output)
+        self.assertEqual(result.exit_code, 0, result.output)
+
+    def test_import_old_local_archives(self):
+        """ Test import of old local archives
+        Expected behavior: Return message in terminal and error code != 0
+        """
+        archives = [('export/migrate/export_v0.1.aiida', '0.1'), ('export/migrate/export_v0.2.aiida', '0.2'),
+                    ('export/migrate/export_v0.3.aiida', '0.3')]
+
+        for archive, version in archives:
+            options = [get_archive_file(archive)]
+            result = self.cli_runner.invoke(cmd_import.cmd_import, options)
+
+            self.assertIsNotNone(result.exception, result.output)
+            self.assertNotEqual(result.exit_code, 0, result.output)
+            self.assertIn(version, result.output, result.exception)
+
+    def test_import_old_url_archives(self):
+        """ Test import of old URL archives
+        Expected behavior: Return message in terminal and error code != 0
+        """
+        url = "https://raw.githubusercontent.com/aiidateam/aiida_core/f685844b290ed23df254873df62da7162bde0f1b/"
+        archive_path = "aiida/backends/tests/fixtures/export/migrate/"
+        archive = 'export_v0.1.aiida'
+        version = '0.1'
+
+        options = [url + archive_path + archive]
+        result = self.cli_runner.invoke(cmd_import.cmd_import, options)
+
+        self.assertIsNotNone(result.exception, result.output)
+        self.assertNotEqual(result.exit_code, 0, result.output)
+        self.assertIn(version, result.output, result.exception)
+
+    @unittest.skip("Reenable when issue #2426 has been solved (migrate exported files from 0.3 to 0.4)")
+    def test_import_url_and_local_archives(self):
+        """Test import of both a remote and local archive
+        TODO: Update 'url' to point at correct commit.
```

Comment:
Is this TODO still relevant and if so what is the 'correct' URL?

Reply:
I will deepen the explanation in the docstring to include the essence of the following:
It is to be changed when the "migration from v0.3 to v0.4" PR has gone through. Now it is referencing the new export files I created for that PR in Sasha's branch. However, as soon as it has been merged, and Sasha probably deletes his branch, then this URL should point to the, essentially same, commit, but in "aiidateam", not "yakutovicha".